dbms/src/Storages/Page/V3/tests/gtest_free_map.cpp

```diff
@@ -427,6 +427,36 @@ TEST_P(SpaceMapTest, TestGetMaxCap)
     }
 }
 
+
+TEST_P(SpaceMapTest, TestGetRightMargin)
+{
```

Comment:
Add a case to cover these lines
https://github.com/pingcap/tiflash/blob/4b398f8ff6bdf52a4b1793f1791e3bc485f11950/dbms/src/Storages/Page/V3/spacemap/SpaceMapSTDMap.h#L116-L119